rosbag:
    args: 
        - "--max-cache-size 100"
        - "-b 0"
benchmark:
    id: 0
    tag: "bench1"
    name: "Bench test"
    overwrite_existing: true
    monitor_frequency: 10
workers:
    - image:
        name: "camera"
        topic: "camera"
        instances: 2
        same_topic: true
        delay: 1000
        max_count: 1000
        size: 1000
        frequency: 100
    - pointcloud2:
        name: "lidar"
        topic: "lidar"
        instances: 1
        same_topic: true
        delay: 1000
        max_count: 1000
        size: 10000
        frequency: 100
    - bytearray:
        name: "ba"
        topic: "ba"
        instances: 10
        same_topic: false
        delay: 1000
        max_count: 1000
        size: 1000
        frequency: 100